src/seedu/addressbook/BetterAddressBook.java

```diff
@@ -0,0 +1,1312 @@
+package seedu.addressbook;
```

Comment:
The whole point of using revision control is that we should not create copies of files. You could have done your changes in the original `AddressBook.java` itself :-)